### ISSUE TITLE
How to deal with Postgres Enum columns? 

### ISSUE THREAD

**Issue by gregsifr** [tiangolo/sqlmodel#31]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import List, Optional
from enum import Enum

import sqlalchemy as sa
import sqlmodel


class Status(str, Enum):
    active = "active"
    banned = "banned"


status_enum = postgresql.ENUM("active", "banned", "c", name="status_enum")


class User(sqlmodel.SQLModel, table=True):
    __tablename__ = "auth_user"
    id: int = sqlmodel.Field(primary_key=True)
    status: Status = sqlmodel.Field(status_enum)
    password: str
```

### Description

How to use Postgres Enums together with SQLModel?

### Operating System

Linux

### SQLModel Version

0.0.3

### Python Version

3.8.11

**Comment by MrMrRobat** [comment]
@gregsifr, could you please reopen the issue for the reasons stated above?

**Comment by gregsifr** [comment]
Apologies, I misunderstood and thought that since there is a PR for it there's no point to keep it open.

**Comment by tiangolo** [comment]
Hey there! I think this was possibly solved in https://github.com/tiangolo/sqlmodel/pull/165. That's available since SQLModel `0.0.7`. Could you confirm?
